3rdparty/jvm/com/fasterxml/jackson/core/BUILD

```diff
@@ -0,0 +1,6 @@
+jvm_artifact(
```

Comment:
I think it would probably be a better example to have all of the `com.fasterxml.jackson` declarations in the same file -- from a maintenance point of view, the versions of these packages are usually kept in lockstep, and having to change a version in multiple files seems obtuse.

Reply:
Yea, there's an interesting tradeoff there right now which I think that we can maybe remove via #13767: it's not possible to share a variable between different BUILD files without a macro. Allowing macros to be declared hierarchically would help to keep these from going out of sync.

But from experience with a thirdparty layout that didn't stick to the `org`/`group` as hierarchy: folks ended up confused about where to put things, and so you ended up with multiple copies of some libraries in parent/child directories. If anything, I'd love to be able to enforce it, although that probably goes too far for casual usage.

I'll ... add a comment for now.

Reply:
It's unfortunate that Jackson has such a fine-grained group structure, which still doesn't make a whole lot of sense to me, tbqh. It really doesn't lend itself well to this multiple files situation.